source/kubernetes/user-access.rst

```diff
@@ -46,6 +46,110 @@ assigned to their account.
 |               | admin namespace.                                                 |
 +---------------+------------------------------------------------------------------+
 
+More information
+----------------
+
+.. raw:: html
+
+   <details>
+   <summary>For a more comprehensive list of the exact permissions that each
+   role gives, you can <a>click here</a> to view a full list for the kubernetes
+   roles.</summary>
+
+.. code-block:: console
+
+   +----------------------+-----------------------------------------------------+
+   | Role                 | Permissions                                         |
+   +======================+=====================================================+
+   | k8s_admin            | container.*                                         |
+   |                      | resourcemanager.projects.get                        |
+   |                      | resourcemanager.projects.list                       |
+   +----------------------+-----------------------------------------------------+
+   | cluster_admin        | container.clusters.create                           |
```

Comment:
I'd suggest removing this cluster_admin role because it's not used at this moment.

Reply:
removed.